examples/basics/src/lib.rs

```diff
@@ -0,0 +1,104 @@
+#[macro_use]
+extern crate neon;
+
+// thanks to @rm-rf-etc for these: https://github.com/neon-bindings/neon/issues/260
+use neon::vm::{Call, JsResult};
+use neon::js::{
+  Object, // <- https://github.com/neon-bindings/neon/issues/57
+  JsFunction,
+  JsObject,
+  JsArray,
+  JsBoolean,
+  JsInteger,
+  JsNull,
+  JsNumber,
+  JsString,
+  JsUndefined};
+
+/*
+	// If you need to inspect the value of an expression:
+	let () = JsBoolean::new();
+*/
+
+// Use _ to squelch warnings, or any name starting with _
```

Comment:
Nice! This kind of contextual tip is such a great way to teach newcomers as they go along.